apps/test/unit/code-studio/pd/professional_learning_landing/landingPageTest.js

```diff
@@ -449,4 +449,371 @@ describe('LandingPage', () => {
 
     sessionStorage.clear();
   });
+
+  describe('Global Edition Configurations', () => {
+    describe('hideMyFacilitatorCenterTab', () => {
```

Comment:
I see there are a lot of test cases, which is great, but it's a bit hard to follow all of them.
If it's not an antipattern, we could consider grouping similar tests for readability such as:

```
describe('Tab Visibility', () => {
  describe('Facilitator Center Tab', () => {
    // Tests for Facilitator Center tab
  });

  describe('Instructor Center Tab', () => {
    // Tests for Instructor Center tab
  });
});

describe('Dialog Visibility', () => {
  describe('Workshop Enrollment Celebration Dialog', () => {
    // Tests for the enrollment dialog
  });
});
```